### PR TITLE
Add a security vulnerability report page

### LINKED ISSUE #14: Add a page listing known and fixed security issues and vulnerabilities.
### Description
We need to have a page to list security issues and vulnerabilities that has been publicly disclosed for eiffel-community projects and repositories

### Motivation
Security and vulnerability information should be easily accessible to contributors and others in the eiffel-community.

### Exemplification
A security researcher finds that a vulnerability affects a project in the eiffel-community. The researcher can quickly check the security page on eiffel-community.github.io to see if a fix for the vulnerability has been released recently.

### Benefits
Easier to get an overview of what has been reported and fixed in the security context on the eiffel-community projects and repositories.

### Possible Drawbacks
None

### PR DESCRIPTION
**Pull request by fredjn**
### Applicable Issues
Fixes #14 

### Description of the Change
Adds a page (security.html) with a list of known security vulnerabilities and the fixes to mitigate them

### Alternate Designs
-

### Benefits
Security considerations today are important and this will show that we, the Eiffel-community, take it seriously.

### Possible Drawbacks
The list may get long and hard to overview

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fredrik Fristedt fredrik.fristedt@axis.com

### CODE REVIEW

**Review by k-hallen-ericsson — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Review by sodero — APPROVED** [review]

**Review by fdegir — APPROVED** [review]

**Comment by henning-roos** [comment]
Is there a good way to see the page as it will appear in browser? It is easier then to review.

**Comment by fredjn** [comment]
> Is there a good way to see the page as it will appear in browser? It is easier then to review.

If you go to the browse to the file in the fork and click the preview-button you get a decent [view](https://refined-github-html-preview.kidonng.workers.dev/fredjn/eiffel-community.github.io/raw/master/security.html).

File: security.html
Comment (henning-roos) [inline security.html:4]: security --> Security

File: security.html
Comment (henning-roos) [inline security.html:39]: Eiffel-community vulnerability and security reports --> Eiffel-community Vulnerability and Security Reports

File: security.html
Comment (henning-roos) [inline security.html:47]: Repository name --> Repository Name

File: security.html
Comment (henning-roos) [inline security.html:48]: Report name --> Report Name

File: security.html
Comment (henning-roos) [inline security.html:41]: Subtitles should also be capitalized according to https://www.constant-content.com/content-writing-service/2008/02/subtitles-should-be-capitalized-just-like-titles/

**Review by henning-roos — CHANGES_REQUESTED** [review]
Maybe a bit too pedantic but I would prefer to have "Title Case" on every header.

File: security.html
Comment (fredjn) [reply security.html]: Fixed

File: security.html
Comment (fredjn) [reply security.html]: Fixed

File: security.html
Comment (fredjn) [reply security.html]: Fixed

File: security.html
Comment (fredjn) [reply security.html]: Fixed

**Review by henning-roos — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

**Comment by fdegir** [comment]
@eiffel-community/eiffel-community-maintainers can we get one more review/approval please?

would be great to be done with the security/vulnerability process before the meeting on Monday.
Thanks.

**Comment by d-stahl-ericsson** [comment]
I'm probably missing something, but shouldn't security.html be linked from somewhere? E.g. the header or the footer.

**Comment by d-stahl-ericsson** [comment]
(Apart from SECURITY.md, that is.)

**Comment by fdegir** [comment]
@fredjn I can't seem to amend this PR to address Daniel's comment to include link to security.html on the page. Can you do it so we merge this and https://github.com/eiffel-community/community/pull/72?

**Comment by fredjn** [comment]
@d-stahl-ericsson - I added security links to header on all subpages, is this along the lines of what you were suggesting?

**Comment by fredjn** [comment]
One strange thing though is that I can't seem to get the changes to render correctly with preview. But I guess it  should be fine.

**Comment by fredjn** [comment]
> One strange thing though is that I can't seem to get the changes to render correctly with preview. But I guess it should be fine.

Works now, seems to have been some indexing och caching issue

**Comment by fredjn** [comment]
> @d-stahl-ericsson - I added security links to header on all subpages, is this along the lines of what you were suggesting?

@d-stahl-ericsson ping!

**Review by d-stahl-ericsson — APPROVED** [review]

**Pull request merged**
